.github/ISSUE_TEMPLATE/launch.md

```diff
@@ -0,0 +1,154 @@
+      hide or block access as needed.
+- [ ] Verify that alias patterns and updates/redirects are working correctly.
+- [ ] Verify that `$settings['hash_salt']` is set.
+- [ ] Verify that `$settings['update_free_access'] = FALSE;` is set.
+- [ ] Verify that `$settings['omit_vary_cookie']` is configured correctly.
+- [ ] Verify that `$settings['file_public_path']` and `file_private_path` is
+      configured correctly.
+- [ ] Confirm that the 404 page is working and styled as expected.
+- [ ] Verify that `$settings['reverse_proxy_*']` configuration is set correctly
+      if a CDN or reverse-proxy is being used.
+
+## Hosting
+- [ ] Determine who will provide the SSL certificate.
+- [ ] Setup HTTPS and test the SSL certificate.
+- [ ] Setup SSL certificate reminders.
+- [ ] Verify that the web server instance size is large enough.
+- [ ] Verify that HTTP/2 is enabled.
+- [ ] Configure redirects from `www` to the base URL (or vice-versa).
+- [ ] Create and test redirect rules.
+- [ ] Create and test reverse-proxy rules.
+- [ ] Verify that asset compression is enabled.
+
+## Infrastructure
+- [ ] Setup database backups in consultation with the devops team.
+- [ ] Create Jenkins production cache clear job if applicable.
+- [ ] Verify that Drupal's `settings.php` and `services.yml` files have the
+      correct read-only permissions.
+- [ ] Confirm syslog visibility and connectivity with external tools.
+- [ ] Verify that errors are configured to be suppressed.
+- [ ] Setup Jenkins jobs for QA (and check that PROD jenkins jobs all running)
+- [ ] Verify that Ansible playbooks follow the latest best practices.
+- [ ] Verify file uploads work.
+- [ ] Verify image styles work.
+- [ ] Run load tests to validate infrastructure.
+- [ ] Consult with the hosting provider for any platform specific launch steps
+      or advice.
+
+## Launch Prep
+- [ ] Create a roll-back plan.
+- [ ] Notify the devops team and confirm they understands what they need to do.
+- [ ] Create ordered checklist and timeline.
+- [ ] Determine who will make the go/no-go call for launch.
+- [ ] Determine who will determine if a roll-back is needed.
+- [ ] Determine who manages DNS and will update the entries for launch.
+- [ ] Lower DNS TTL to ensure quick propagation on launch day.
+- [ ] Verify that `robots.txt` is configured to allow traffic to the production
+      site (i.e. remove any changes that were made for development).
+- [ ] Verify that users of the site are aware of the cutover time/plan.
+- [ ] Create a calendar invite with a video call link that can be shared with
+      everyone who is involved with the launch.
+- [ ] Verify that user accounts have been migrated or recreated, and that users
+      have access they need.
+
+## Site Launch
+- [ ] Disable HTTP Authorization if applicable.
+- [ ] Update DNS entries.
+- [ ] Test the site for egregious issues.
+
+## Post Launch
+- [ ] Monitor site log for errors.
```

Comment:
Which logs are we talking about here?

Reply:
Whatever ones we deem important? I could say PHP logs, but I kind of like this being vague. Thoughts?

Reply:
I worry that it’s vague enough to gloss over, but I don’t feel too strongly about this. I’m cool with leaving it as is.